internal/store/postgres/user_repository.go

```diff
@@ -276,6 +282,23 @@ func (r UserRepository) List(ctx context.Context, flt user.Filter) ([]user.User,
 		flt.Page = defaultPage
 	}
 
+	var (
+		orderExpr     exp.OrderedExpression
+		orderedColumn = goqu.C("email")
```

Comment:
should the default be `id` instead of `email`?

Reply:
sure we could sort based on id by default